src/js/utils/attribute-token-list.js

```diff
@@ -0,0 +1,95 @@
+export class AttributeTokenList {
+  #el;
+  #attr;
+  #tokens = [];
+
+  constructor(el, attr) {
+    this.#el = el;
+    this.#attr = attr;
+  }
+
+  [Symbol.iterator]() {
+    return this.#tokens.values();
+  }
+
+  get length() {
+    return this.#tokens.length;
+  }
+
+  get value() {
+    return this.#tokens.join(' ') ?? '';
+  }
+
+  set value(val) {
+    if (val === this.value) return;
+    this.#tokens = [];
+    this.add(...(val?.split(' ') ?? []));
+  }
+
+  toString() {
+    return this.value;
+  }
+
+  item(index) {
+    return this.#tokens[index];
+  }
+
+  values() {
+    return this.#tokens.values();
+  }
+
+  keys() {
+    return this.#tokens.keys();
+  }
+
+  forEach(callback) {
+    this.#tokens.forEach(callback);
+  }
+
+  add(...tokens) {
+    tokens.forEach((t) => {
+      if (!this.contains(t)) this.#tokens.push(t);
```

Comment:
Should we be case-insensitive? Perhaps with a `t.toLowerCase()` here?

Thinking about `<media-controller hotkeys="noarrowleft">`, since `event.key` would be `ArrowLeft`, and might make it nicer if we allowed `noF` for clarity.

Reply:
we're effectively case-insensitive based on your "early bail" logic in the handler.

Reply:
The difference is if I do `mc.hotKeys.add('noArrowLeft')`. Then, it won't turn things off.

Reply:
I don't feel strongly on this generally, but I think we should "bias for action" and ship what we have, since we can add your suggestion as a followup in a non-breaking way.